recipes/s5cmd/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set name = "s5cmd" %}
+{% set version = "1.4.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/peak/s5cmd/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: b756ac8c69175d04e972c3b4deb6b689986d4260fbe83e32caea52d1061995a0
+
+build:
+  script: go install -v .
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('go') }}
+
+test:
+  commands:
+    - s5cmd --help
+
+about:
+  home: https://github.com/peak/{{ name }}
+  summary: 's5cmd is a very fast S3 and local filesystem execution tool.'
+  description: |
+    s5cmd is a very fast S3 and local filesystem execution tool.
+    It comes with support for a multitude of operations including tab completion and wildcard support for files,
+    which can be very handy for your object storage workflow while working with large number of files.
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE
```

Comment:
You need to bundle the licenses of the statically linked libraries, see https://github.com/conda-forge/conda-forge.github.io/issues/1052

Reply:
Thanks for taking a look, @xhochy. I've added all the licenses --- let me know if this format is suitable. One small note, I've adopted the MIT license for https://github.com/termie/go-shutil/blob/master/LICENSE (based on their message).